pkgs/misc/cups/drivers/hl3140cw/default.nix

```diff
@@ -0,0 +1,76 @@
+{stdenv, fetchurl, cups, dpkg, gnused, makeWrapper, ghostscript, file, a2ps, coreutils, gawk}:
+
+let
+  version = "1.1.4-0";
+cupsdeb = fetchurl {
```

Comment:
The indentation is all messed up in this file, this should be indented.

Reply:
Sorry about that, I carelessly copied most of this from an existing expression which I think had tabs.